src/plugins_exts_yangdata.h

```diff
@@ -0,0 +1,28 @@
+/**
+ * @file plugins_exts_yangdata.h
+ * @author Radek Krejci <rkrejci@cesnet.cz>
+ * @brief ietf-yang-yangdata API
+ *
+ * Copyright (c) 2021 CESNET, z.s.p.o.
+ *
+ * This source code is licensed under BSD 3-Clause License (the "License").
+ * You may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://opensource.org/licenses/BSD-3-Clause
+ */
+
+#ifndef LY_PLUGINS_EXTS_YANGDATA_H_
+#define LY_PLUGINS_EXTS_YANGDATA_H_
+
+#include "tree_schema.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+#ifdef __cplusplus
+}
+#endif
+
+#endif /* LY_PLUGINS_EXTS_YANGDATA_H_ */
```

Comment:
There will be some future use for this, I guess? Otherwise I do not really understand the use.

Reply:
actually no, there was use, but later it became useless - removing it.